script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -1141,7 +1141,7 @@ I don't know what form the enemy has taken this time―――
 [sha:cd1b60fa05253b58a5be6917973b1c4e0ed0923e]{
 -- Page 374, Offset 25211. Glued.
 -- 男性なのか女性なのか、子供なのか大人なのかさえ判らない」
-whether they're a man or a woman, or even if they're a child or an adult."
+whether he's a man or a woman, or even if he's a child or an adult."
```

Comment:
I like 'they're' better than 'he's' in this sentence.

Reply:
Agree

Reply:
Hmm so change "He" in the line before it?

Reply:
> so I haven't even had a chance to see **his** current form yet.